images/ceph/cephfs_provisioner.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python
```

Comment:
Where is this script triggered? I don't see it called anywhere.

Reply:
In ```images/ceph/Dockerfile```,  ```images/ceph/cephfs_provisioner.py``` is copy to ```/usr/local/bin/cephfs_provisioner```.
triggered at ```cmd := exec.Command(provisionCmd, args...)``` in provisioner.go